test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/DistributedCacheTagHelperTest.cs

```diff
@@ -272,6 +272,29 @@ public void UpdateCacheEntryOptions_SetsAbsoluteExpiration_IfExpiresOnIsSet()
         }
 
         [Fact]
+        public void UpdateCacheEntryOptions_DefaultsTo30SecondsSliding_IfNoEvictionCriteriaIsProvided()
```

Comment:
Can you add a test where the priority is specified (or just amend this to include it)/